.gitignore

```diff
@@ -96,3 +96,6 @@ odoo.tar.gz
 odoo-9.0
 odoo-10.0
 odoo-11.0
+
```

Comment:
this should stay in your global .gitignore :wink:

Reply:
:+1: I learnt something today... Thanks!